style.css

```diff
@@ -45,15 +45,9 @@ body {
 	background-color: lightgray;
 	z-index: 1;
 	position: relative;
-	display: none;
```

Comment:
In case like this scrollbar for history container should appear:
![image](https://user-images.githubusercontent.com/11320052/168143850-71f1d24e-526b-4b7d-a090-5a43b57132a1.png)

Or maybe you want to have for example only last 10 actions? Scrollbar preferred.